### ISSUE TITLE
Create "As featured in" section in the about subpage

### ISSUE THREAD

**Issue by ChucklesOnGitHub** [open-neuroscience/open-neuroscience.github.io#35]
I think it would be great to link the articles in which the project has been featured to the webpage.
A nice way to do this is have a link side-by-side with an image from the article with a brief caption about the article and media.

**Examples:**
Link: https://www.simonsfoundation.org/2021/01/05/the-democratization-of-neuroscience/
Title: The Democratization of Neuroscience
Picture: timeline or top banner
Brief: article written by Ashley Juavinett for the Simons Foundation webpage describing the origins and growing landscape of open tools in neuroscience, including hardware, software and training.

Link: https://www.nature.com/articles/d41586-020-03193-5
Title. How DIY technologies are democratizing science
Picture: TReND or DIY hardware
Brief: technology feature article by Sandeep Ravindran for Nature research journal on how leveraging open science and 3D printing to build equipment and produce reagents in-house can help close to the technological gap and democratize research.

**Comment by matiasandina** [comment]
I like this! What about adding this as an image slider or table?
Probably harder to implement than text but looks good.

**Comment by ChucklesOnGitHub** [comment]
In all truth I self-assigned this to practice creating content for the webpage, but I need help to start.
I was thinking the format for "project cards" that is default with Academic would work for this section. It shows the Picture and has the title and brief, and links to the actual article.
As for where to put it, it could be in the About section we have now or on a new page).

TL;DR: I worked on the content and it is ready, I need help with the implementation.

**Comment by ChucklesOnGitHub** [comment]
Add this piece https://arxiv.org/ftp/arxiv/papers/2106/2106.15852.pdf

**Comment by amchagas** [comment]
Link: https://arxiv.org/ftp/arxiv/papers/2106/2106.15852.pdf
Title: Harnessing the Benefits of Open Electronics in Science
Picture: Figure 2?
Brief: Preprint highlighting easy to use electronics and their impact on the democratization of science. The preprint also highlights resources curating open science projects.

**Comment by amchagas** [comment]
working on implementing this... a bit of trouble getting the images to load...
